locale/en.po

```diff
@@ -3773,34 +3773,43 @@ msgstr "Save failed"
 msgid "ERROR_SAVING"
 msgstr "Error Saving"
 
-#: ../src/saving/NewSaveMenu.cs:74
-msgid "CHOSEN_FILENAME_ALREADY_EXISTS"
+#: ../src/saving/NewSaveMenu.cs:78
+msgid "THE_CHOSEN_FILENAME_ALREADY_EXISTS"
 msgstr "The chosen filename ({0}) already exists. Overwrite?"
 
-#: ../src/saving/NewSaveMenu.cs:172
+#: ../src/saving/NewSaveMenu.cs:112
+msgid "ATTEMPT_TO_WRITE_FILE_FAILED"
+msgstr ""
+"The attempt to write this save file has failed. The save name may be too long. And maybe you don't have write access to the save folder / file."
```

Comment:
Line is too long, there should be line breaks created automatically here if you use Poedit. Also probably could be worded like:
```suggestion
"Attempt to write this save file has failed. The save name may be too long or you may not have write access to the save folder/file."
```

Reply:
Actually I just have switched back to not using line wrapping. This time it seems that weblate agrees with that. So finally the translations update PRs won't have a bunch of back and forth with the line wrapping.